src/amo/sagas/landing.js

```diff
@@ -52,8 +44,7 @@ export function* fetchLandingAddons({
       api,
       filters: {
         ...filters,
-        featured: enableFeatureRecommendedBadges ? undefined : true,
```

Comment:
Can we also remove the support for `featured` in the API? I am thinking about:

- https://github.com/mozilla/addons-frontend/blob/44511f71fb472c7de1ad37632c3c13ff545c3f2f/src/core/searchUtils.js#L23
- https://github.com/mozilla/addons-frontend/blob/44511f71fb472c7de1ad37632c3c13ff545c3f2f/src/core/api/search.js#L22
- https://github.com/mozilla/addons-frontend/blob/44511f71fb472c7de1ad37632c3c13ff545c3f2f/src/core/api/search.js#L49

Reply:
See my comment about about this for my reasoning, but I have reconsidered and think it would be a good idea to remove this.